packages/@sanity/schema/src/preview/guessPreviewConfig.js

```diff
@@ -39,7 +39,8 @@ function resolveFileAssetPath(typeDef) {
   return fieldWithFileAsset ? `${fieldWithFileAsset.name}.asset` : undefined
 }
 
-export default function guessPreviewFields(objectTypeDef) {
+export default function guessPreviewFields(rawObjectTypeDef) {
+  const objectTypeDef = {fields: [], ...objectTypeDef}
```

Comment:
Shouldn't this be `...rawObjectTypeDef`?

Reply:
whoa 🙈 yes it should

Reply:
Fixed